app/models/concerns/hyrax/solr_document_behavior.rb

```diff
@@ -75,7 +75,7 @@ def work?
     end
 
     # Method to return the model
-    def hydra_model(classifier: ActiveFedora.model_mapper)
+    def hydra_model(classifier: Hyrax.config.default_model_mapper)
       first('has_model_ssim')&.safe_constantize ||
         classifier.classifier(self).best_model
```

Comment:
What happens if default_model_mapper isn't set, which appears to happen if disable_wings is true?